app/src/main/res/layout-land/activity_login.xml

```diff
@@ -171,16 +171,37 @@
 
                 </LinearLayout>
 
-                <fr.free.nrw.commons.ui.widget.HtmlTextView
-                    android:id="@+id/about_privacy_policy"
-                    style="?android:textAppearanceSmall"
-                    android:layout_below="@id/buttonFrame"
-                    android:layout_width="wrap_content"
+                <LinearLayout
+                    android:id="@+id/linksFrame"
+                    android:layout_width="match_parent"
                     android:layout_height="wrap_content"
+                    android:layout_below="@id/buttonFrame"
                     android:layout_marginBottom="@dimen/standard_gap"
-                    android:layout_gravity="center_horizontal"
-                    android:text="@string/about_privacy_policy"
-                    android:layout_centerHorizontal="true"/>
+                    android:layout_marginEnd="@dimen/standard_gap"
+                    android:layout_marginLeft="@dimen/standard_gap"
+                    android:layout_marginRight="@dimen/standard_gap"
+                    android:layout_marginStart="@dimen/standard_gap">
+
+                    <fr.free.nrw.commons.ui.widget.HtmlTextView
+                        android:id="@+id/reset_password_link"
+                        style="?android:textAppearanceSmall"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginLeft="@dimen/large_gap"
+                        android:layout_weight="1"
+                        android:layout_marginBottom="@dimen/standard_gap"
+                        android:text="@string/reset_password_link" />
+
+                    <fr.free.nrw.commons.ui.widget.HtmlTextView
+                        android:id="@+id/about_privacy_policy"
+                        style="?android:textAppearanceSmall"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_weight="1"
```

Comment:
You should add `android:gravity="center_horizontal"` line to make texts centered.